utils/tail/tail.go

```diff
@@ -133,15 +123,24 @@ func (t *Tail) watchFile() error {
 		return err
 	}
 	defer t.file.Close()
-	err = t.watcher.Watch(t.filename)
+
+	watchDir := filepath.Dir(t.filename)
+	err = t.watcher.AddWatch(watchDir, inotify.IN_MOVED_FROM|inotify.IN_DELETE)
```

Comment:
Are we sure that a rename is what logrotation does? Sounds most likely, but just checking..